src/.env.development

```diff
@@ -0,0 +1,90 @@
+CLIENT_NAME="Skyra"
+CLIENT_VERSION="6.0.0-dev [Sapphire Edition]"
+CLIENT_PREFIX="sd!"
+CLIENT_REGEX_PREFIX=""
+CLIENT_OWNERS=""
+CLIENT_ID=""
+CLIENT_SHARDS="auto"
```

Comment:
There's a property d.js uses automatically, you should use that instead

Reply:
... which property?